implementations/duktape/src/lib.rs

```diff
@@ -0,0 +1 @@
+extern crate c_path;
```

Comment:
this is just requiring the c_path crate in the main. basically, `lib.rs` is where all the rust will go when we convert everything.